docs/ldml/tr35-keyboards.md

```diff
@@ -2030,6 +2043,316 @@ Footnotes:
 
 * * *
 
+## Keyboard Test Data
+
+Keyboard Test Data allows the keyboard author to provide regression test data to validate the repertoire and behavior of a keyboard. Tooling can run these regression tests against an implementation, and can also be used as part of the development cycle to validate that keyboard changes do not deviate from expected behavior.
+
+Test data files have a separate DTD, named `ldmlKeyboardTest.dtd`.  Sample test data files are located in the `keyboards/test` subdirectory.
```

Comment:
We should add some wording about best practice of grouping and naming of tests, multiple files per keyboard possible, multiple tests per file.

Think this may help head off a future quagmire of randomly organised keyboard tests...

Reply:
Not sure what the best practices should be at the moment.  This seems somewhat like a repository policy discussion, which we need to have more discussions around.  Right now I have a `3.0/` directory with keyboards, and a `test/` directory.